internal/server/server.go

```diff
@@ -304,3 +320,36 @@ func UpdateServerTags(service *service.Service, serverUUID string, oldTags, newT
 
 	return AddNewServerTags(service, serverUUID, newTags)
 }
+
+// ValidateServerAndStorageBackupRules checks if the server and (to be) attached storages have conflicting backup rules.
+func ValidateServerAndStorageBackupRules(service *service.Service, d *schema.ResourceData) error {
+	if _, ok := d.GetOk("simple_backup"); ok {
```

Comment:
personally would reverse the check here so we immediately return nil in this case (so we don't have so many blocks/so much indentation)

Reply:
Removed this whole function. It could lead to errors on subsequent updates with some configs.